addons/60fps/addon.json

```
@@ -28,7 +33,7 @@
   ],
   "settings": [
     {
-      "name": "Alt+GreenFlag FPS",
+      "name": "FPS",
```

Almost forgot, I think clarifying that this is the custom framerate you can set might not hurt.
```suggestion
      "name": "Custom frames per second (FPS)",
```

---

"Custom frames per second" sounds a little weird to say compared to "custom FPS" (because nouns),
```suggestion
      "name": "Custom FPS",
```
but saying the full thing instead of just the abbreviation sounds like a good thing. Maybe we can say "frames per second" somewhere in the description or a note instead of here if that feels better?